manual/manual/refman/exten.etex

```diff
@@ -1792,6 +1792,19 @@ the attributes are considered to apply to the payload:
 fun%foo[@bar] x -> x + 1 === [%foo (fun x -> x + 1)[@bar ] ];
 \end{verbatim}
 
+Quoted strings "{|...|}" are particularly interesting for payloads
+that embed foreign syntax fragments. Those fragments can be interpreted
+by a preprocessor and turned into OCaml code without requiring escaping
+quotes. For instance, you can use "[%sql {|...|}]" to
+represent arbitrary SQL statements -- assuming you have a ppx-rewriter
+that recognizes the "%sql" extension.
+
+Note that the non-extension form, for example "{sql|...|sql}", should
+not be used for this purpose. Indeed, the user cannot see in the code that
```

Comment:
I'd be even more explicit and say "...not be used for signaling an extension is in use. Indeed..."

Reply:
Good idea.